src/api/buildQuery.js

```diff
@@ -190,6 +190,21 @@ const queries = {
     }
   },
 
+  deleteImage: (input) => {
+    return {
+      template: `
+        mutation DeleteImage($input: DeleteImageInput!) {
+          deleteImage(input: $input) {
```

Comment:
@ingalls also exposed a `deleteImages` mutation that takes an array of `imageIds`. Seeing as we will want to be able to delete more than one image at a time we might as well just always use (passing in an array w/ a single ID if we just want to delete one at a time) that rather than having two separate queries?

```
{
    query:`
        mutation DeleteImages($input: DeleteImagesInput!){
            deleteImages(input: $input) {
                message
            }
        }
    `,
    variables:{
        input: {
            imageIds: ['project:123']
        }
    }
}
```

Reply:
ok, that's changed now